bpe/apply.py

```diff
@@ -45,7 +45,7 @@ def run(self):
         lexicon.load(lexicon_path)
 
         lm_tokens = set()
-        for l in lexicon.lemma:
+        for l in lexicon.lemmas:
```

Comment:
can we please call it by the Greek plural `lemmata`

Reply:
Using the English plural is also accepted by the Oxford dictionary.